src/transaction-runner.coffee

```diff
@@ -174,6 +175,15 @@ class TransactionRunner
     @logs ?= []
     for log in results.logs or []
       @logs.push log
+
+    # For unknown reasons, the sandboxed code in Pitboss runs in "future"
+    # on Windows (and just sometimes - it's flaky). The extreme case is
+    # that sandboxed 'before' hooks can have timestamps with a millisecond
+    # later time then the HTTP transaction itself. Following line
+    # synchronizes the time. It waits until the time of the normal Node.js
+    # runtime happens to be later than time inside the Pitboss sandbox.
+    while Date.now() - results.now < 0 then # ...then do nothing...
```

Comment:
Dirty hack but should work, would not a sleep as in `setTimeout` work better. Depends on how out of sync the time is though.